readme.md

```diff
@@ -1,3 +1,70 @@
 # instructions for all assignments and exercises
 
-Instructions will be released as they are developed
+Instructions will be released as they are developed.
+
+## Assignment Submission Summary
+Assignments are submitted in two parts:
+1. Push your changes to GitHub.
+2. Verify that your changes are showing on GitHub, then copy and paste the URL into the assignment text box on D2L.
+
+## Git and GitHub Summary
+Words in `<brackets>` are just placeholders - do not type the `<>`.
+### To start an assignment
+1. To start your assignment, click on the GitHub Classroom link in D2L and click "accept this assignment". Refresh the page to get your unique assignment URL.
+2. On your computer, run the command `git clone <url>`, where `<url>` is your assignment URL (do not type the `<>`).
+
+### Working on an assignment
+3. Make sure you are working in your assignment directory, either through Git Bash or VS Code's integrated terminal.
+4. Pull changes from GitHub, just in case:
+   ```
+   git pull
+   ```
+5. Make edits to your text file(s). Don't forget to save changes!
+6. Add/commit/push changes:
+   1. First, add the file(s) you want to commit to the staging area:
+        ```
+        git add <filename>
+        ```
+   2. Then, commit all the files with a message describing the changes you made:
+        ```
+        git commit -m "<message>"
+        ```
+   3. Push all your commits to GitHub
+        ```
+        git push
+        ```
+> You can add/commit/push as many times as you like up until you submit your assignment
+
+### Final submission
+7. When you are happy with the state of your code, add/commit/push one last time and go to your assignment repository URL. You can find this by clicking on the "GitHub Classroom link" for the assignment in D2L, or from the GitHub homepage.
```

Comment:
I'm a bit "ehhhhhh" about re-doing the Classroom link - they gotta jump through some additional hoops there, so maybe just mention the homepage route?

Reply:
Good point, I actually had no idea students were doing this until I saw it happening. Updated :)